tests/test_ops.py

```diff
@@ -1,2 +1,59 @@
-""" Empty file for now
-"""
+import pytest
+
+from devito import Grid, TimeFunction, Operator, Eq, configuration
+from devito.ir.iet import FindNodes, Expression, printAST
+
+pytestmark = pytest.mark.skipif(configuration['backend'] != 'ops',
+                                reason="'ops' wasn't selected as backend on startup")
+
+
+class TestOpsSyntax(object):
+    """
+        Check if the ops syntax is being written correctly.
+    """
+
+    def test_ops_declarations(self):
+        """
+            Creates a simple one dimension grid with 0's and add 1's to the grid.
+        """
+        grid = Grid(shape=10)
+
+        u = TimeFunction(name='u', grid=grid)
+
+        eq = Eq(u.forward, u + 1.0)
+
+        op = Operator(eq)
+
+        # Checks ops init declaration
+        assert 'ops_init(0,NULL,1);' in str(op)
+        # Checks ops_exit declaration
+        assert 'ops_exit();' in str(op)
+
+
+class TestOpsExpression(object):
+    """
+        Tests OPS generated expressions.
+        Some basic tests for the generated expression with ops.
+    """
+    @pytest.mark.parametrize('expr, expected', [
```

Comment:
as I wrote on Slack, I'd like to see many more test cases. You can/should also compare to stringyfied sympy expressions, not to IET objects